src/components/List.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from "classnames";
+import { nullValuesAsUndefined } from "null-as-undefined";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/core/_lists.scss";
+
+/**
+ * The types of {@link List}.
+ */
+export enum ListTypes {
+  Bullet = "bullet",
+  Number = "number"
+}
+
+/**
+ * The proptypes for {@link List}.
+ */
+export interface ListProps {
+  id?: string | null;
+  className?: string | null;
+  /**
+   * Defaults to an unordered list if no type is specified.
+   */
+  type?: ListTypes | null;
+  /**
+   * The items in the list. Can be anything from a string to another list
+   * component.
+   */
+  items: React.ReactNode[];
+}
+
+/**
+ * A list of items.
+ */
+export const List: React.FunctionComponent<ListProps> = (
+  props: ListProps
+): JSX.Element => {
+  const { id, className, type, items } = nullValuesAsUndefined(props);
+
+  const ListType = type == ListTypes.Number ? "ol" : "ul";
+
+  return (
+    <ListType
+      id={id}
+      className={classNames(
+        "govuk-list",
+        "lbh-list",
+        { "lbh-list--number govuk-list--number": type == ListTypes.Number },
+        { "lbh-list--bullet govuk-list--bullet": type == ListTypes.Bullet },
```

Comment:
Isn't this actually `type !== ListType.Number` to match the above behaviour? What happens if `type` is undefined?

Reply:
If it's undefined, neither of the classes get added in. There's a scenario where we just have a plain list with no styling on it. It would be hard to test correctly without some sort of visual testing really.

Reply:
So if it's undefined, the list is a bullet list, but without the classes that style it correctly? That feels wrong to me, and should definitely be documented if we really want it that way.

Reply:
It's an unordered list without any markers if not defined. I'm basing it off the implementation for lists at https://lbh-frontend.herokuapp.com/examples/typography

Reply:
Ok. Let's be really clear about the difference, then.